### PR TITLE
Upload data by pages

### PR DESCRIPTION
**Pull request by IvanJanuary**

### CODE REVIEW

File: NetworkRequest/NetworkRequest/Controller/IssueViewController.swift [hunk 1]
```diff
@@ -9,12 +9,13 @@ import UIKit
 
 class IssueViewController: UIViewController, UISearchBarDelegate {
     
-    var buttonMore = UIButton()
     var searchText: String = ""
     var page = 1
+    var isLoading = true
```
Comment (Lana-Resquit) [inline NetworkRequest/NetworkRequest/Controller/IssueViewController.swift]: [Question] Why do you need this variable here and in other controllers?

File: NetworkRequest/NetworkRequest/Controller/IssueViewController.swift [hunk 2]
```diff
@@ -95,5 +84,14 @@ extension IssueViewController: UITableViewDataSource, UITableViewDelegate {
         cell.detailTextLabel?.text = issue.url ?? "No description available"
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        if (indexPath.row + 1) % 10 == 0  && indexPath.row == (page * 10) - 1 {
+            print("indexPath: \(indexPath.row + 1)")
```
Comment (Lana-Resquit) [inline NetworkRequest/NetworkRequest/Controller/IssueViewController.swift]: [Minor] This print is not needed anymore ( here and in other controllers)

File: NetworkRequest/NetworkRequest/Controller/IssueViewController.swift (hunk 1 above)
Comment (IvanJanuary) [reply NetworkRequest/NetworkRequest/Controller/IssueViewController.swift]: fixed

File: NetworkRequest/NetworkRequest/Controller/IssueViewController.swift (hunk 2 above)
Comment (IvanJanuary) [reply NetworkRequest/NetworkRequest/Controller/IssueViewController.swift]: remove too

**Pull request merged**